profab/role/ami/ubuntu.py

```diff
@@ -16,7 +16,7 @@ class Configure(Role):
     def ami(self, region, bits, size):
         """Return the AMI that was passed in to the role.
         """
-        disk = 'ebs' if size == 't1.micro' else 'instance'
```

Comment:
This old code is wrong, but I'm not sure the new code is better. Is always defaulting to EBS the right thing?

Reply:
"Is always defaulting to EBS the right thing?"
In that time, they hard coded to make it work with our UAT environment.
UAT instance is R4.xlarge